### ISSUE TITLE
Warning in Optimal_transport_reconstruction_2

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#2111]
Recent version of the testsuite show a warning in Optimal_transport_reconstruction_2 with the Debian-Testing platform:
```
[...]/include/CGAL/Optimal_transportation_reconstruction_2.h:992:19: warning: variable 'cv' set but not used [-Wunused-but-set-variable]
     Vertex_handle cv = vmap[v];
                   ^~
```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.10-I-192/Optimal_transportation_reconstruction_2/TestReport_lrineau_Debian-Testing.gz

The issue is in CGAL-4.10-beta.

**Comment by afabri** [comment]
In case there was no need for this line, the function `copy_neighbors()` also needs no `vmap`.

**Comment by afabri** [comment]
@sgiraudot can you have a look please.

**Comment by lrineau** [comment]
Fixed by @sgiraudot in #2117.
